goIRCBot/src/functions/functions.go

```diff
@@ -0,0 +1,53 @@
+package functions
+
+func About(name string) (string, string, string, string, string, string, string) {
+	val1:="Hello @"+name+", Welcome to the Open Source Developers Group @ IIIT - H"
+	val2:="Mailing List : https://groups.google.com/forum/?fromgroups#!forum/iiit-osdg"
+	val3:="Blog : http://iiitosdg.wordpress.com/"
+	val4:="IRC : Well, you guys are already here aren't you :P"
+	val5:="GitHub : https://github.com/OSDG-IIITH/"
+	val6:="Want to get a project forked under the github group? Register it at http://osdg.iiit.ac.in/github/"
+	val7:="Doing GSoC this summer? Check out http://osdg.iiit.ac.in/gsoc15/"
```

Comment:
Why does it point to that page with the text "Doing GSoC this summer?" ?

I think a better way to introduce that page could be "People who did GSoC last summer: " or something else that sounds better than that example :P